users/views.py

```diff
@@ -72,3 +75,25 @@ def post(self, request):
         
         except DataError:
             return JsonResponse({"message": "DATA_ERROR"}, status=400)
+
+class UserDetailView(View):
+    @ConfirmUser
+    def get(self, request):
+        wish_list  = [{
+            "name" : restaurant.name,
+            "address" : restaurant.address,
+            "sub_category" : restaurant.sub_category.name,
+            "average_rating" : Review.objects.filter(restaurant_id=restaurant.id).aggregate(Avg("rating"))["rating__avg"] 
+            if Review.objects.filter(restaurant_id=restaurant.id) else 0,
+            "is_wished" : True,
+            "food_image" : Image.objects.filter(food__restaurant__id=restaurant.id).first().image_url
```

Comment:
restaurant 객체로부터 접근해서 만들 수 있지 않을까요?

```suggestion
            "food_image" : restaurant...
```

Reply:
아하 알겠습니다! 
"갖고 오고자 하는 데이터에 맞는 모델을 불러와야 흐름이 안 망가진다"라고 말씀하셨던 부분을 적용해봤는데 살짝 기준이 헷갈리기는 합니다! ㅋㅋ 수정했습니다!